### DISCUSSION TITLE
No buildings on win10/xp11 (overlay).

### DISCUSSION THREAD

**Discussion by SamPro910** [kubilus1/autoortho#232] [Q&A] [answered]
Basically what the title says. All the buildings are flat (but satellite imagery is visible). I thought it was to do with load order: i saw on a site that Overlays should be above Global Scenery. However even after this modification, the buildings are nowhere to be seen.

Something caught my eye though. When running X-Plane, the `scenery_packs.ini` puts `z_eur_XX` on the top (00-15), and removes `z_ao_eur`. Is my install wrong?

**Comment by kubilus1** [comment]
The 0.5.x ver will have a names like 'z_ao_eur' while the older release will have the numbered dirs.  You just need to make sure you have overlays set above the scenery.

**Reply by SamPro910** [reply]
Ah, roger. Still don't have buildings though, I could try more things tomorrow (well actually today but read afternoon Z+3)

**Comment by dpdurst** [comment]
This is from XP11 but you'll get the picture.. lol the Ortho stuff goes at the very bottom, Same for XP12. You'll see flat buildings and such but all your normal FS buildings and scenery should be fine

SCENERY_PACK Custom Scenery/Global Airports/
SCENERY_PACK Custom Scenery/yAutoOrtho_Overlays/
SCENERY_PACK Custom Scenery/z_ao_na/
SCENERY_PACK Custom Scenery/z_autoortho/

**Reply by SamPro910** [reply]
Yes, this was my load order initally. But the autogen still does not work.

**Reply by dpdurst** [reply]
Then you either still have something not correct or you have another issue. It may be helpful to post your scenery.ini and autoortho log file after you have selected the airport and have no buildings. Also a screenshot so we can see would help too.

**Reply by SamPro910** [reply]
Roger, installing the beta now, trying to see if that would work

**Reply by dpdurst** [reply]
Cool, post up your logs and a screenshot so we can see if it acts the same

**Reply by SamPro910** [reply]
[scenery_packs.ini](https://github.com/kubilus1/autoortho/files/11897394/scenery_packs.txt)
[XP Log.txt](https://github.com/kubilus1/autoortho/files/11897400/Log.txt)
[autoortho.log](https://github.com/kubilus1/autoortho/files/11897422/autoortho.log)
![image](https://github.com/kubilus1/autoortho/assets/79455895/818f66ac-f438-4950-9e23-885cc65c0485)

This is with the beta

**Comment by dpdurst** [comment]
Cool thanks, I'm loading this up in XP11 and testing as well hold tight

**Comment by dpdurst** [comment, marked as answer]
Ok so I have first checked out where you were LIRN in Italy.. I loaded up the XP scenery and then loaded up that airport screenshot below.
Then I loaded up the EUR AO scenery and screenshot below with all buildings in both default and using AO

I compared your AO log to mine and no difference. 
I compared your log.txt to mine before AO and after and I see your missing some scenery area's which I am not. 
Might want to run the XPInstaller.exe and check to see if fines any missing scenery and double check the EUR scenery.

0:06:32.938 I/DSF: Warning for missing scenery file: +39+014
0:06:32.938 I/DSF: Warning for missing scenery file: +39+012
0:06:32.938 I/DSF: Warning for missing scenery file: +39+013

Ah wait, take your scenery.ini and open it up, find these line 

SCENERY_PACK Custom Scenery/Global Airports/
SCENERY_PACK Custom Scenery/KSEA Demo Area/
SCENERY_PACK Custom Scenery/LOWI Demo Area/

And move them so its configured like this and see if this changes things also

SCENERY_PACK Custom Scenery/X-Plane Landmarks - Washington DC/
SCENERY_PACK Custom Scenery/KSEA Demo Area/
SCENERY_PACK Custom Scenery/LOWI Demo Area/
SCENERY_PACK Custom Scenery/Global Airports/


I don't see AO causing this issue at the moment.

LIRN default XP scenery
![LIRN default](https://github.com/kubilus1/autoortho/assets/8261471/e4ef9b1c-8444-45ce-aa0d-70fd68883c3d)


LIRN with AO 
![LIRN ao](https://github.com/kubilus1/autoortho/assets/8261471/cb93bc31-5cf2-4463-a3ee-0590fbe810b2)

**Comment by SamPro910** [comment]
I haven't tested, but @dpdurst has confirmed that this issue is not with AO, so the discussion can be closed solved.

**Comment by Pilotmk3** [comment]
I am also getting same problem with no buildings , plese someone can help me with it 
[Log.txt](https://github.com/kubilus1/autoortho/files/12775225/Log.txt)
![20230930230800_1](https://github.com/kubilus1/autoortho/assets/141163254/3afc4409-5de0-4e00-a89e-6bdb3759ff0a)

**Reply by Pilotmk3** [reply]
and here is log file 
[Log.txt](https://github.com/kubilus1/autoortho/files/12775236/Log.txt)

airport - VARK (rajkot india)

**Reply by Pilotmk3** [reply]
and scenery.ini here 
[scenery.txt](https://github.com/kubilus1/autoortho/files/12775243/scenery.txt)
